src/patternfly/components/Page/page.scss

```diff
@@ -407,17 +411,53 @@ $pf-c-page--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl",
       }
 
       &.pf-m-no-padding#{$breakpoint-name} {
-        padding: 0;
+        --pf-c-page__main-section--PaddingTop: 0;
+        --pf-c-page__main-section--PaddingRight: 0;
+        --pf-c-page__main-section--PaddingBottom: 0;
+        --pf-c-page__main-section--PaddingLeft: 0;
       }
     }
   }
 }
 
 .pf-c-page__main-wizard {
   flex: 1 0 0;
+  background-color: var(--pf-c-page__main-wizard--BackgroundColor);
   border-top: var(--pf-c-page__main-wizard--BorderTopWidth) solid var(--pf-c-page__main-wizard--BorderTopColor);
 }
 
+.pf-c-page__main-nav,
+.pf-c-page__main-breadcrumb,
+.pf-c-page__main-section,
+.pf-c-page__main-wizard {
+  &.pf-m-limit-width {
+    display: flex;
+    flex-direction: column;
```

Comment:
would this have an impact on the elements that are inside of these containers ? what if they had two images in a flex row?

Reply:
Nope, all of the children of the `pf-c-page__main-[nav/section/breadcrumb/etc]` would be put in the `main-body` element, so no other children would be part of that flex layout. Though I just updated the PR to remove this flex styling and added `height: 100%` to the `main-body` element, which seems a bit safer. The main goal is just that if someone puts something with `height: 100%` in one of these sections, and applies `.pf-m-limit-width`, their thing will be 100% height - which means the `main-body` element also needs to be 100% height.